azuredevops/internal/service/git/resource_git_repository.go

```diff
@@ -43,7 +43,9 @@ func ResourceGitRepository() *schema.Resource {
 		Read:   resourceGitRepositoryRead,
 		Update: resourceGitRepositoryUpdate,
 		Delete: resourceGitRepositoryDelete,
-
+		Importer: &schema.ResourceImporter{
```

Comment:
Both https://dev.azure.com/{organization}/_apis/git/repositories/{repositoryId}?api-version=6.0
https://dev.azure.com/{organization}/{project}/_apis/git/repositories/{repositoryId}?api-version=6.0 can used to get the repository, the only difference is if we set the projectName/ID in the URL. As the API document have explicate the request parameter but not marked can be ignored, we should  set the projectName/ID in the request URL  [get repository](https://docs.microsoft.com/en-us/rest/api/azure/devops/git/repositories/get%20repository?view=azure-devops-rest-5.1). Suggest change the import format from `terraform import azuredevops_git_repository.repository 00000000-0000-0000-0000-000000000000` to ``terraform import azuredevops_git_repository.repository projectName/00000000-0000-0000-0000-000000000000`, you can replace the importer function with ```tfhelper.ImportProjectQualifiedResource()```.